reactions/include/Copter/HALO.h

```diff
@@ -5,12 +5,11 @@
 
 // limits on mass exponents
 const double Mmax = 18.;
-const double Mmin = 8.;
+const double Mmin = 5.;
 
 class HALO {
 public:
-  HALO(const Cosmology& C, const PowerSpectrum& P_l, real epsrel = 1e-4);
-
+  HALO(const Cosmology& C, const PowerSpectrum& P_l, const PowerSpectrum& P_cb,const PowerSpectrum& P_nu, real epsrel = 1e-4);
```

Comment:
Will this work if someone wants massless neutrinos?

Reply:
Yes, in the massless case the user just needs to supply the linear spectrum for total matter in place of P_cb and P_nu. This should be added to the readme file. I've added a comment in HALO.h and HALO.cpp